src/system/types/cosmere.ts

```diff
@@ -51,6 +51,15 @@ export enum Attribute {
     Presence = 'pre',
 }
 
+export enum AttributeShortLabel {
+    Strength = 'COSMERE.Actor.Attribute.Strength.short',
+    Speed = 'COSMERE.Actor.Attribute.Speed.short',
+    Intellect = 'COSMERE.Actor.Attribute.Intellect.short',
+    Willpower = 'COSMERE.Actor.Attribute.Willpower.short',
+    Awareness = 'COSMERE.Actor.Attribute.Awareness.short',
+    Presence = 'COSMERE.Actor.Attribute.Presence.short',
+}
```

Comment:
Is there a reason to have these as an enum? If we don't need to re-use them, I think we should just include them in the configuration like all other labels.